### PR TITLE
[DPS-1151 ] Fixes to documentation 

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
Changes described here: https://livechatinc.atlassian.net/browse/DPS-1511

### CODE REVIEW

File: content/messaging/agent-chat-api/rtm-reference/index.mdx [hunk 1]
```diff
@@ -1228,9 +1228,9 @@ It returns [summaries](#chat-summaries) of the chats an Agent has access to.
...
 | `filters.properties.<namespace>.<name>.<filter_type>` | No       | `any`    |                                                                                             |
-| `pagination`                                          | No       | `object` |                                                                                             |
-| `pagination.page`                                     | No       | `number` | Default: 1, minimum: 1, maximum: 1000                                                       |
-| `pagination.total`                                    | No       | `number` | The total number of threads that match filters; default: 25, min: 0, max: 100.              |
+| `order`                                               | No       | `string` | Possible values: `asc` - oldest chats firstm `desc` - newest chats first (default)          |
+| `limit`                                               | No       | `number` | Defaul: 10, maximum: 100                                                                    |
```
Comment (lpmichalik) [inline content/messaging/agent-chat-api/rtm-reference/index.mdx]: 'default'

File: content/messaging/agent-chat-api/index.mdx [hunk 2]
```diff
@@ -1230,9 +1230,9 @@ It returns [summaries](#chat-summaries) of the chats an Agent has access to.
...
 | `filters.group_ids`                                   | No       | `array`  | Array of group IDs.                                                                         |
 | `filters.properties.<namespace>.<name>.<filter_type>` | No       | `any`    |                                                                                             |
-| `pagination`                                          | No       | `object` |                                                                                             |
-| `pagination.page`                                     | No       | `number` | Default: 1, minimum: 1, maximum: 1000                                                       |
-| `pagination.total`                                    | No       | `number` | Total number of threads that match filters; default: 25, minimum: 0, maximum: 100.          |
+| `order`                                               | No       | `string` | Possible values: `asc` - oldest chats firstm `desc` - newest chats first (default)          |
```
Comment (litwinow) [inline content/messaging/agent-chat-api/index.mdx]:
```suggestion
| `order`                                               | No       | `string` | Possible values: `asc` - oldest chats first, `desc` - newest chats first (default)          |
```

File: content/messaging/agent-chat-api/rtm-reference/index.mdx [hunk 3]
```diff
@@ -1228,9 +1228,9 @@ It returns [summaries](#chat-summaries) of the chats an Agent has access to.
...
 | `filters.group_ids`                                   | No       | `array`  | Array of group IDs.                                                                         |
 | `filters.properties.<namespace>.<name>.<filter_type>` | No       | `any`    |                                                                                             |
-| `pagination`                                          | No       | `object` |                                                                                             |
-| `pagination.page`                                     | No       | `number` | Default: 1, minimum: 1, maximum: 1000                                                       |
-| `pagination.total`                                    | No       | `number` | The total number of threads that match filters; default: 25, min: 0, max: 100.              |
+| `order`                                               | No       | `string` | Possible values: `asc` - oldest chats firstm `desc` - newest chats first (default)          |
```
Comment (litwinow) [inline content/messaging/agent-chat-api/rtm-reference/index.mdx]:
```suggestion
| `order`                                               | No       | `string` | Possible values: `asc` - oldest chats first, `desc` - newest chats first (default)          |
```

File: content/messaging/agent-chat-api/v3.2/index.mdx [hunk 4]
```diff
@@ -1210,9 +1210,9 @@ It returns [summaries](#chat-summaries) of the chats an Agent has access to.
...
 | `filters.group_ids`                                   | No       | `array`  | Array of group IDs.                                                                         |
 | `filters.properties.<namespace>.<name>.<filter_type>` | No       | `any`    |                                                                                             |
-| `pagination`                                          | No       | `object` |                                                                                             |
-| `pagination.page`                                     | No       | `number` | Default: 1, minimum: 1, maximum: 1000                                                       |
-| `pagination.total`                                    | No       | `number` | Total number of threads that match filters; default: 25, minimum: 0, maximum: 100.          |
+| `order`                                               | No       | `string` | Possible values: `asc` - oldest chats firstm `desc` - newest chats first (default)          |
```
Comment (litwinow) [inline content/messaging/agent-chat-api/v3.2/index.mdx]:
```suggestion
| `order`                                               | No       | `string` | Possible values: `asc` - oldest chats first, `desc` - newest chats first (default)          |
```

File: content/messaging/customer-chat-api/v3.2/index.mdx [hunk 5]
```diff
@@ -2791,6 +2806,41 @@ Here's what you need to know about **webhooks**:
...
+| --------------------------- | ----------------------------- | ------------------------------------------------------------------ |
+| `authentication`            | Authentication error          | An invalid or expired access token.                                |
+| `authorization`             | Authorization error           | Agent is not allowed to perform the action.                        |
+| `entity_too_large`          | Upload limit exceeded (10MB). | Client's request is too large.                                     |
+| `internal`                  | Internal server error         |                                                                    |
+| `license_expired`           | License expired               | The end of license trial or subcription.                           |
+| `license_not_found`         | License not found             | License with the specified ID doesn't exist.                       |
+| `misdirected_request`__*__  | Wrong region                  | Client's request should be performed to another region.            |
```
Comment (litwinow) [inline content/messaging/customer-chat-api/v3.2/index.mdx]: what's this `__*__`? in some other file I've seen `**/***` next to `misdirected_request`

**Pull request merged**